internal/builders/generic/README.md

```diff
@@ -270,3 +277,70 @@ generated as an [in-toto](https://in-toto.io/) statement with a SLSA predicate.
   }
 }
 ```
+
+## Integration With Other Build Systems
+
+This section explains how to generate non-forgeable SLSA provenance with existing build systems.
+
+### Provenance for Goreleaser
+
+If you use [Goreleaser](https://github.com/goreleaser/goreleaser-action) to generate your build, you can easily
```

Comment:
nit: GoReleaser seems to refer to itself with a capital 'R' in their docs.

Reply:
good catch, thanks